.github/workflows/dart.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        sdk: [2.12.4, stable, beta, dev]
+        sdk: ['2.12.4', 'stable', 'beta', 'dev']
```

Comment:
Shouldn't we still be able to handle these if they're unquoted? I thought yaml would only convert the "numberlike" strings to numbers.

Reply:
Those 4 should all come through as-is (no magic promotion by us would be necessary). My idea here was to encourage users to quote all their values here, so we're not generally in a place of having to guess what they might have typed.

Reply:
I think I'll remove the unnecessary quotes - we do handle the auto-promotion case, and it looks cleaner w/o the quotes.